engine/assets/render/tone_mapping.fs

```diff
@@ -5,12 +5,162 @@ in vec2 fragUv;
 uniform sampler2D hdrTexture;
 uniform float gamma;
 uniform float exposure;
+uniform uvec2 screenSize;
+
+const float EDGE_THRESHOLD_MIN = 0.0312;
+const float EDGE_THRESHOLD_MAX = 0.125;
+const float SUBPIXEL_QUALITY = 0.75;
+const int ITERATIONS = 12;
```

Comment:
If these parameters (or some of them) are things you need to tinker with to get right, I think you should pass them as an uniform, so that you can have these values come from a resource or component on the CPU side!

Then we could have a tool to configure that, which would be cool :eyes:
In this case, ig it would go into the `ToneMapping` component? Since there are so many uniforms already, I would put them all in a single uniform block, as seen in other shaders

One other extra that I think would be really cool would be to have a separate `FXAA` component, managed by the tone mapping plugin.
If the component is present, then `FXAA` is enabled, and the settings are fetched from it. Otherwise, we just have plain old tonemapping